doc/computation.rst

```diff
@@ -99,6 +99,48 @@ These operations automatically skip missing values, like in pandas:
 If desired, you can disable this behavior by invoking the aggregation method
 with ``skipna=False``.
 
+Rolling window operations
+=========================
+
+``DataArray`` objects include a :py:meth:`~xarray.DataArray.rolling` method. This
+method supports rolling window aggregation:
+
+.. ipython:: python
+
+    x = xr.DataArray(np.random.random((3, 5)))
+    x
+
+:py:meth:`~xarray.DataArray.rolling` is applied along one dimension using the
+name of the dimension as a key (e.g. ``dim_1``) and the window size as the value
```

Comment:
nit: again, I prefer using meaningful dimensions names in example -- it encourages best practices :)